src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -38,8 +38,11 @@ def handle_unverified_request
        when "branch?" then "branch"
        else exception.query
     end
-    message = "Sorry, you are not authorized to " +
-              (exception.message ? "#{pundit_action} this #{exception.record.class}." : "perform this action.")
+    if pundit_action && exception.record
+      message = "Sorry, you are not authorized to #{pundit_action} this #{exception.record.class}."
```

Comment:
I don't see why we would ever should show such a message. What would a user do with knowing that a say NoMethodError happened.

Either he / she is not authorized or there was an internal error. In the latter case we should just say "Sorry there was an internal error. Please try again."